command/agent.go

```diff
@@ -121,6 +121,18 @@ Policy Options:
     The default evaluation interval that will be applied to all scaling policies
     which do not specify an evaluation interval.
 
+Policy Evaluation Options:
+
+  -policy-eval-ack-timeout=<dur>
+    The time limit that an eval must be ACK'd before being considered NACK'd.
+
+  -policy-eval-delivery-limit=<num>
+    The maximum number of times a policy evaluation can be dequeued from the broker.
+
+  -policy-eval-workers=<key:value>
+    The number of workers to initialize for each queue, formatted as
+    <queue1>:<num>,<queue2>:<num>.
```

Comment:
I'm curious if it would be worth detailing the possible queues here so operators can easily discover utilising this flag?

Reply:
I added to the the [docs](https://github.com/hashicorp/nomad/pull/10183), but you are right, it's good to have it here as well.